lib/ffmpeg/transcoderbase.py

```diff
@@ -174,7 +176,8 @@ def check_output(self):
 
   @timefn("ffmpeg")
   def call_ffmpeg(self, iopts, oopts):
-    return call("ffmpeg -v verbose {} {}".format(iopts, oopts))
+    return call(f"{exe2os('ffmpeg')}"
+                " -v verbose {} {}".format(iopts, oopts))
 
   def transcode(self):
     self.validate_caps()
```

Comment:
In the "for loop", below, we dump the transcoded/encoded files to "yuv" files via ffmpeg command.  Thus, we need to use a filepath2os on each yuv file.

Also, in `check_resolution(...)`, we call `ffmpeg_probe_resolution(...)` which will need the encoded/transcoded filename via filepath2os.

Reply:
Updated. Pls review it again, thanks!